<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="afficheText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>381</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>230</width>
      <height>24</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>162</width>
      <height>198</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Police :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButtonArial">
        <property name="text">
         <string>Arial</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButtonBlackletter">
        <property name="text">
         <string>Courier</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButtonTime">
        <property name="text">
         <string>Times new roman</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QCheckBox" name="checkBoxItalique">
      <property name="text">
       <string>Italique</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxGras">
      <property name="text">
       <string>Gras</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionouvrir"/>
    <addaction name="actionenregistrer"/>
    <addaction name="actiona_propos"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionouvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionenregistrer">
   <property name="text">
    <string>enregistrer</string>
   </property>
  </action>
  <action name="actiona_propos">
   <property name="text">
    <string>a propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxItalique</sender>
   <signal>stateChanged(int)</signal>
   <receiver>afficheText</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxGras</sender>
   <signal>stateChanged(int)</signal>
   <receiver>afficheText</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonArial</sender>
   <signal>clicked()</signal>
   <receiver>afficheText</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonBlackletter</sender>
   <signal>clicked()</signal>
   <receiver>afficheText</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonTime</sender>
   <signal>clicked()</signal>
   <receiver>afficheText</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
